### PR TITLE
Issue 52610. Fix 'prefer_interpolation_to_compose_strings' when 'toString()' with arguments.

### PR DESCRIPTION
**Pull request by scheglov**
Fixes # [(52610)](https://github.com/dart-lang/sdk/issues/52610)

### CODE REVIEW

**Review by srawlins — APPROVED** [review]
+1 thanks for the test conversion!

**Review by pq — COMMENTED** [review]
Nice!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # 3.1.0-wip
 
 - new lint: `no_wildcard_variable_uses`
+- fix `prefer_interpolation_to_compose_strings` for false-positive when
```
Comment (pq) [inline CHANGELOG.md:4]: FWIW, I think we're headed away from CHANGELOG entries for fixes of this scope. 

See: https://github.com/dart-lang/linter/pull/4430

Comment (scheglov) [reply CHANGELOG.md:4]: Ah, OK, I will keep this in mind for the future.

**Pull request merged**